src/recommendation/src/main/scala/RecommendationHelper.scala

```diff
@@ -33,6 +40,95 @@ object SparkHelper {
 
 trait RecommendationParams extends ALSParams
 
+trait RecommendationModelParams extends Params with ALSModelParams with HasPredictionCol {
```

Comment:
it looks like this holds not only params but an interface - I would change the name of the trait then to make it more explicit that this isn't only for params.